### ISSUE TITLE
Autocrosspostbot Crossposts the same post from different communities

### ISSUE THREAD

**Issue by JamesNinelives** [Toldry/RedditAutoCrosspostBot#17]
Essentially the same post but from different communities (r/aww and r/boottoobig), both crossposted to /r/pottedcats.

https://www.reddit.com/r/pottedcats/comments/mpw711/do_not_disturb_me_i_am_blooming/

https://www.reddit.com/r/pottedcats/comments/mjzpga/stocks_are_high_and_business_is_booming/

**Comment by Toldry** [comment]
Yeah, the image is slightly different (note the white background with the text above). 

The bot's repost detecing capabilities are not advanced enough for these cases sadly
